plan.py

```diff
@@ -33,6 +46,46 @@
 saver = tf.train.Saver()
 saver.restore(sess, 'model/model.ckpt')
 
+if args.use_openvino:
+    import subprocess
+    import sys
+    import mo_tf
+
+    from openvino.inference_engine import IECore
+
+    # Save frozen graph
+    pb_model_path = 'model/model.pb'
+    # output_node_names = ['Softmax_1']
+    output_node_names = ['TopKV2']
+    input_node_name = ['Placeholder', 'Placeholder_2']
+    graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph_def,
+            output_node_names)
+
+    from tensorflow.python.tools import optimize_for_inference_lib
+    graph_def = optimize_for_inference_lib.optimize_for_inference(
+            graph_def, input_node_name, output_node_names, tf.float32.as_datatype_enum)
+
+    with tf.io.gfile.GFile(pb_model_path, 'wb') as f:
+        f.write(graph_def.SerializeToString())
+
+    # Convert to OpenVINO IR
+    subprocess.run(
+        [
+            sys.executable, mo_tf.__file__, '--input_model', pb_model_path,
+            '--output_dir', 'model/', '--input', 'Placeholder,Placeholder_2', '--input_shape', "[1, 10000],[1]",
+            '--freeze_placeholder_with_value', "Placeholder_2->5"
```

Comment:
5?

Reply:
We should get top 5 predictions

Reply:
Can we avoid this hardcoded number? Perform TopK in python (so output of network will be Softmax_1) or pass by input?